dev/TabView/TabView.idl

```diff
@@ -44,13 +47,41 @@ unsealed runtimeclass TabView : Windows.UI.Xaml.Controls.ListView
 
     event Windows.Foundation.TypedEventHandler<TabView, Object> AddButtonClick;
```

Comment:
Also AddButtonClickCommand (maybe should be feedback on the spec?)

Reply:
Added AddButtonCommand and AddButtonCommandParameter